### DISCUSSION TITLE
Deployment Tag is not visible

### DISCUSSION THREAD

**Discussion by manikmagar** [BattlesnakeOfficial/feedback#232] [Q&A]
Hi!

I have DataWeave, Hungry, MuleSoft tags on my snake. On "My Battlesnakes" page, I see these icons as expected -

![image](https://user-images.githubusercontent.com/877286/219971845-c04e33d4-9cef-44f3-8bb3-d9ce81ecdb7b.png)

But I am not seeing MuleSoft as Deployment cloud on Leaderboards or ranking lists - 

![image](https://user-images.githubusercontent.com/877286/219971929-45ba7a3e-8b69-49fa-aa67-a0686eeeece4.png)

Am I missing anything there? Thanks in advance for taking time to see this.

Regards,
Manik

**Comment by alexandramartinez** [comment]
FYI - since the UI changed, this question doesn't apply anymore.
